CommunityToolShedMVC/CommunityToolShedMVC/ViewModels/BorrowToolViewModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace CommunityToolShedMVC.ViewModels
+{
+    public class BorrowToolViewModel
+    {
+        [Required]
+        public int ToolId { get; set; }
```

Comment:
It's not necessary to decorate value type properties with the `Required` attribute (they're required by default).